src/lava/proc/conv/models.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import numpy as np
+
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+from lava.magma.core.model.py.ports import PyInPort, PyOutPort
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.resources import CPU
+from lava.magma.core.decorator import implements, requires, tag
+from lava.magma.core.model.py.model import PyLoihiProcessModel
+from lava.proc.conv.process import Conv
+
+from lava.proc.conv import utils
+
+
+class AbstractPyConvModel(PyLoihiProcessModel):
```

Comment:
Is this an abstract class? If not, please remove the 'Abstract' from the name.

Reply:
Yes it's an abstract class

Reply:
I was rather asking how it is an abstract class as I didn't see any methods that needed to be implemented by subclasses? Am I missing something?